config.yml

```diff
@@ -21,7 +21,7 @@ defaults:
       type: 'oauth'
 
   webhook:
-    secret: !env WEBHOOK_SECRET
+    secret: !env:list WEBHOOK_SECRETS
```

Comment:
I think we should either make this plural everywhere ("secrets") or singular everywhere ("secret").  The advantage to plural is that it make it clear it's an array.  The advantage to singular is that the existing env variable will work without change.